nimbus-primitives/src/lib.rs

```diff
@@ -60,6 +60,11 @@ where
 	}
 }
 
+/// Return current block author without passing any input
+pub trait GetAuthor<AccountId> {
+	fn get_author() -> AccountId;
+}
```

Comment:
`frame_support::traits::FindAuthor` takes input `digests` so I created this trait to expose the block author stored directly in the author inherent pallet

Reply:
Actually, I prefer just using `Get<T::AccountId>` to introducing this new trait

Will make this change now